test/unit/io_managers_test.py

```diff
@@ -249,3 +255,232 @@ def test_error_when_reading_view_without_metadata(pudl_sqlite_io_manager_fixture
     input_context = build_input_context(asset_key=AssetKey(asset_key))
     with pytest.raises(ValueError):
         pudl_sqlite_io_manager_fixture.load_input(input_context)
+
+
+def test_ferc_xbrl_sqlite_io_manager_dedupes(mocker, tmp_path):
+    db_path = tmp_path / "test_db.sqlite"
+    datapackage = json.dumps(
+        {
+            "profile": "tabular-data-package",
+            "name": "test_db",
+            "title": "Ferc1 data extracted from XBRL filings",
+            "resources": [
+                {
+                    "path": f"sqlite:///{db_path}",
```

Comment:
* I don't think a `sqlite:///` path is valid in a datapackage, at least not until using Frictionless v5.
* According to the spec I don't think these can have `tabular-data-package` or `tabular-data-resource` profiles because it's not referring to either CSV or inline JSON data.

Reply:
This is a fake datapackage that only exists to mimic the structure of a datapackage descriptor and hold the primary keys, so I'll just make this a "csv" and add a comment to that effect here.

Separately, was looking at the spec & I guess all datapackages [MUST](https://specs.frictionlessdata.io/data-package/#descriptor) be named `datapackage.json`? I guess we're out of spec there, too.

Reply:
Yeah, there's that too...

We should eventually be able to annotate all of the outputs -- multiple SQLite DBs and parquet files -- with a single datapackage.json that refers to the individual tables in all of the different files as resources.

But good to switch it to CSVs for now in this testing setup.